doc/releases/release-notes-3.5.rst

```diff
@@ -223,6 +223,9 @@ Trusted Firmware-M
 Networking
 **********
 
+* Wi-Fi
+  * Added Passive scan support. Scan API signature changed.
```

Comment:
This are 2 topics, right?

The people that will read would like know what changed on the scan API and the why.

For instance: _The WIFI scan API was updated with WIFI scan parameter to allow scan mode selection._

Reply:
Yes, it should have been there. Updated as per suggestion. Thanks